src/main/java/core/basesyntax/service/strategy/handler/PurchaseHandler.java

```
@@ -0,0 +1,14 @@
+package core.basesyntax.service.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseHandler implements OperationHandler {
+    @Override
+    public void process(FruitTransaction transaction) {
+        if (Storage.storage.get(transaction.getFruit()) > transaction.getQuantity()) {
```

What happens if condition is false? Should you throw any exception?